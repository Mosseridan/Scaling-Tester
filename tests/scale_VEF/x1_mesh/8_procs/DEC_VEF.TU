Statistiques de post resolution

Temps total                       1.10877

