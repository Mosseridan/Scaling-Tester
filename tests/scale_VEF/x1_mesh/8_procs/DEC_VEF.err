Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF
feenableexcept enabled.
Data file : DEC_VEF
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF/x1_mesh/8_procs
Nom du cas DEC_VEF
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
Reading of data for a Schema_euler_explicite time scheme
Reading of the maximum number of time steps
The next backup, by security, will take place after 23 hours of calculation.
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block Cavite1
Step of 3D mesh in progress... 
It was created 33% of coordxIt was created 66% of coordx
It was created 10% of coordyIt was created 20% of coordyIt was created 30% of coordyIt was created 40% of coordyIt was created 50% of coordyIt was created 60% of coordyIt was created 70% of coordyIt was created 80% of coordyIt was created 90% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary paroi_a_20
End of reading of the boundary : paroi_a_20
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary Paroi_echange1
End of reading of the boundary : Paroi_echange1
Reading of the boundary Devant1
End of reading of the boundary : Devant1
Reading of the boundary Derriere1
End of reading of the boundary : Derriere1
Reading of the block Cavite2
Step of 3D mesh in progress... 
It was created 14% of coordxIt was created 28% of coordxIt was created 42% of coordxIt was created 57% of coordxIt was created 71% of coordxIt was created 85% of coordx
It was created 33% of coordyIt was created 66% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary Paroi_echange1
End of reading of the boundary : Paroi_echange1
Reading of the boundary paroi_a_40
End of reading of the boundary : paroi_a_40
Reading of the boundary paroi_a_20
End of reading of the boundary : paroi_a_20
Reading of the boundary Devant2
End of reading of the boundary : Devant2
Reading of the boundary Derriere2
End of reading of the boundary : Derriere2
16 double nodes were found 
Compressing objects of type Zone in progress... 
agglomeration of Zones Cavite1 and Cavite2
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_solide
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block Cavite3
Step of 3D mesh in progress... 
It was created 14% of coordxIt was created 28% of coordxIt was created 42% of coordxIt was created 57% of coordxIt was created 71% of coordxIt was created 85% of coordx
It was created 14% of coordyIt was created 28% of coordyIt was created 42% of coordyIt was created 57% of coordyIt was created 71% of coordyIt was created 85% of coordy
It was created 33% of coordzIt was created 66% of coordz
Step of mesh ended 
Reading of the boundary Paroi_echange2
End of reading of the boundary : Paroi_echange2
Reading of the boundary Entree
End of reading of the boundary : Entree
Reading of the boundary Paroi
End of reading of the boundary : Paroi
Reading of the boundary Paroi_echange2
End of reading of the boundary : Paroi_echange2
Reading of the boundary Sortie
End of reading of the boundary : Sortie
Reading of the boundary Paroi
End of reading of the boundary : Paroi
Reading of the boundary Devant3
End of reading of the boundary : Devant3
Reading of the boundary Derriere3
End of reading of the boundary : Derriere3
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_fluide
Execution of the interpreter Tetraedriser_homogene_compact
Association of the domain on which the interpreter applies Tetraedriser_homogene_compact
 NB ELEM : 153 NB NODE : 288
Construction of the OctreeRoot OK 
We have split the cubes...
Splitting of the boundaries
Splitting of the connectors
Splitting of the internal faces
END of Tetraedriser_homogene_compact...
  1 NbElem=3672  NbNod=1011
Execution of the interpreter Tetraedriser_homogene_compact
Association of the domain on which the interpreter applies Tetraedriser_homogene_compact
 NB ELEM : 147 NB NODE : 256
Construction of the OctreeRoot OK 
We have split the cubes...
Splitting of the boundaries
Splitting of the connectors
Splitting of the internal faces
END of Tetraedriser_homogene_compact...
  1 NbElem=3528  NbNod=935
Decouper : Splitting of a domain
 Domain name to split : dom_solide
==============================================
The extreme coordinates of the domain dom_solide are:
x is between 0 and 1
y is between 0 and 1
z is between 0 and 1
==============================================
==============================================
The volume cells of the domain dom_solide are:
sum(volume cells)= 0.51
mean(volume cells)= 0.000138889
min(volume cells)= 0.000138889
max(volume cells)= 0.000138889
==============================================
domaine = dom_solide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 279
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 No correction to be made
The partitioner has generated 8 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
  Number of nodes    : 157
  Number of elements : 456
  Number of joints   : 3
  Joint 0 PeVoisin 1 NbSommets 27 NbFaces 36 NbElemDist 178
  Joint 1 PeVoisin 4 NbSommets 10 NbFaces 8 NbElemDist 84
  Joint 2 PeVoisin 5 NbSommets 26 NbFaces 34 NbElemDist 206
               Total:    NbSommets 63 NbFaces 78 NbElemDist 468
Writing part 0 into the binary file DOM1_0000.Zones
Writing of 157 nodes.
Writing of 456 elements.
 Construction of part number 1
  Number of nodes    : 153
  Number of elements : 460
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 27 NbFaces 36 NbElemDist 210
  Joint 1 PeVoisin 2 NbSommets 28 NbFaces 40 NbElemDist 196
  Joint 2 PeVoisin 4 NbSommets 8 NbFaces 7 NbElemDist 50
  Joint 3 PeVoisin 5 NbSommets 3 NbFaces 1 NbElemDist 8
               Total:    NbSommets 66 NbFaces 84 NbElemDist 464
Writing part 1 into the binary file DOM1_0001.Zones
Writing of 153 nodes.
Writing of 460 elements.
 Construction of part number 2
  Number of nodes    : 151
  Number of elements : 460
  Number of joints   : 2
  Joint 0 PeVoisin 1 NbSommets 28 NbFaces 40 NbElemDist 192
  Joint 1 PeVoisin 3 NbSommets 25 NbFaces 36 NbElemDist 180
               Total:    NbSommets 53 NbFaces 76 NbElemDist 372
Writing part 2 into the binary file DOM1_0002.Zones
Writing of 151 nodes.
Writing of 460 elements.
 Construction of part number 3
  Number of nodes    : 148
  Number of elements : 460
  Number of joints   : 1
  Joint 0 PeVoisin 2 NbSommets 25 NbFaces 36 NbElemDist 180
               Total:    NbSommets 25 NbFaces 36 NbElemDist 180
Writing part 3 into the binary file DOM1_0003.Zones
Writing of 148 nodes.
Writing of 460 elements.
 Construction of part number 4
  Number of nodes    : 151
  Number of elements : 458
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 10 NbFaces 8 NbElemDist 63
  Joint 1 PeVoisin 1 NbSommets 8 NbFaces 7 NbElemDist 64
  Joint 2 PeVoisin 5 NbSommets 30 NbFaces 41 NbElemDist 252
  Joint 3 PeVoisin 7 NbSommets 26 NbFaces 36 NbElemDist 161
               Total:    NbSommets 74 NbFaces 92 NbElemDist 540
Writing part 4 into the binary file DOM1_0004.Zones
Writing of 151 nodes.
Writing of 458 elements.
 Construction of part number 5
  Number of nodes    : 149
  Number of elements : 460
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 26 NbFaces 34 NbElemDist 179
  Joint 1 PeVoisin 1 NbSommets 3 NbFaces 1 NbElemDist 24
  Joint 2 PeVoisin 4 NbSommets 30 NbFaces 41 NbElemDist 220
  Joint 3 PeVoisin 7 NbSommets 6 NbFaces 4 NbElemDist 25
               Total:    NbSommets 65 NbFaces 80 NbElemDist 448
Writing part 5 into the binary file DOM1_0005.Zones
Writing of 149 nodes.
Writing of 460 elements.
 Construction of part number 6
  Number of nodes    : 148
  Number of elements : 460
  Number of joints   : 1
  Joint 0 PeVoisin 7 NbSommets 25 NbFaces 36 NbElemDist 172
               Total:    NbSommets 25 NbFaces 36 NbElemDist 172
Writing part 6 into the binary file DOM1_0006.Zones
Writing of 148 nodes.
Writing of 460 elements.
 Construction of part number 7
  Number of nodes    : 151
  Number of elements : 458
  Number of joints   : 3
  Joint 0 PeVoisin 4 NbSommets 26 NbFaces 36 NbElemDist 178
  Joint 1 PeVoisin 5 NbSommets 6 NbFaces 4 NbElemDist 26
  Joint 2 PeVoisin 6 NbSommets 25 NbFaces 36 NbElemDist 172
               Total:    NbSommets 57 NbFaces 76 NbElemDist 376
Writing part 7 into the binary file DOM1_0007.Zones
Writing of 151 nodes.
Writing of 458 elements.
End of the interpreter Decouper
Decouper : Splitting of a domain
 Domain name to split : dom_fluide
==============================================
The extreme coordinates of the domain dom_fluide are:
x is between 0.3 and 1
y is between 0.3 and 1
z is between 0 and 1
==============================================
==============================================
The volume cells of the domain dom_fluide are:
sum(volume cells)= 0.49
mean(volume cells)= 0.000138889
min(volume cells)= 0.000138889
max(volume cells)= 0.000138889
==============================================
domaine = dom_fluide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 332
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 5
The partitioner has generated 8 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
 Adding of a new joint : 5
 Adding of a new joint : 6
  Number of nodes    : 146
  Number of elements : 442
  Number of joints   : 4
  Joint 0 PeVoisin 4 NbSommets 28 NbFaces 39 NbElemDist 404
  Joint 1 PeVoisin 5 NbSommets 0 NbFaces 0 NbElemDist 50
  Joint 2 PeVoisin 6 NbSommets 0 NbFaces 0 NbElemDist 19
  Joint 3 PeVoisin 7 NbSommets 22 NbFaces 29 NbElemDist 250
               Total:    NbSommets 50 NbFaces 68 NbElemDist 723
Writing part 0 into the binary file DOM2_0000.Zones
Writing of 146 nodes.
Writing of 442 elements.
 Construction of part number 1
 Adding of a new joint : 7
  Number of nodes    : 147
  Number of elements : 444
  Number of joints   : 4
  Joint 0 PeVoisin 3 NbSommets 4 NbFaces 2 NbElemDist 84
  Joint 1 PeVoisin 4 NbSommets 18 NbFaces 22 NbElemDist 234
  Joint 2 PeVoisin 5 NbSommets 27 NbFaces 36 NbElemDist 371
  Joint 3 PeVoisin 7 NbSommets 0 NbFaces 0 NbElemDist 6
               Total:    NbSommets 49 NbFaces 60 NbElemDist 695
Writing part 1 into the binary file DOM2_0001.Zones
Writing of 147 nodes.
Writing of 444 elements.
 Construction of part number 2
 Adding of a new joint : 4
  Number of nodes    : 147
  Number of elements : 440
  Number of joints   : 5
  Joint 0 PeVoisin 3 NbSommets 28 NbFaces 40 NbElemDist 392
  Joint 1 PeVoisin 4 NbSommets 0 NbFaces 0 NbElemDist 30
  Joint 2 PeVoisin 5 NbSommets 21 NbFaces 25 NbElemDist 250
  Joint 3 PeVoisin 6 NbSommets 18 NbFaces 22 NbElemDist 290
  Joint 4 PeVoisin 7 NbSommets 17 NbFaces 17 NbElemDist 311
               Total:    NbSommets 84 NbFaces 104 NbElemDist 1273
Writing part 2 into the binary file DOM2_0002.Zones
Writing of 147 nodes.
Writing of 440 elements.
 Construction of part number 3
 Adding of a new joint : 7
  Number of nodes    : 147
  Number of elements : 440
  Number of joints   : 4
  Joint 0 PeVoisin 1 NbSommets 4 NbFaces 2 NbElemDist 78
  Joint 1 PeVoisin 2 NbSommets 28 NbFaces 40 NbElemDist 406
  Joint 2 PeVoisin 5 NbSommets 16 NbFaces 18 NbElemDist 236
  Joint 3 PeVoisin 7 NbSommets 0 NbFaces 0 NbElemDist 10
               Total:    NbSommets 48 NbFaces 60 NbElemDist 730
Writing part 3 into the binary file DOM2_0003.Zones
Writing of 147 nodes.
Writing of 440 elements.
 Construction of part number 4
 Adding of a new joint : 2
  Number of nodes    : 146
  Number of elements : 440
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 28 NbFaces 39 NbElemDist 378
  Joint 1 PeVoisin 1 NbSommets 18 NbFaces 22 NbElemDist 309
  Joint 2 PeVoisin 2 NbSommets 0 NbFaces 0 NbElemDist 55
  Joint 3 PeVoisin 5 NbSommets 17 NbFaces 20 NbElemDist 308
  Joint 4 PeVoisin 7 NbSommets 14 NbFaces 13 NbElemDist 223
               Total:    NbSommets 77 NbFaces 94 NbElemDist 1273
Writing part 4 into the binary file DOM2_0004.Zones
Writing of 146 nodes.
Writing of 440 elements.
 Construction of part number 5
 Adding of a new joint : 0
 Adding of a new joint : 6
  Number of nodes    : 143
  Number of elements : 440
  Number of joints   : 7
  Joint 0 PeVoisin 0 NbSommets 0 NbFaces 0 NbElemDist 40
  Joint 1 PeVoisin 1 NbSommets 27 NbFaces 36 NbElemDist 381
  Joint 2 PeVoisin 2 NbSommets 21 NbFaces 25 NbElemDist 348
  Joint 3 PeVoisin 3 NbSommets 16 NbFaces 18 NbElemDist 299
  Joint 4 PeVoisin 4 NbSommets 17 NbFaces 20 NbElemDist 278
  Joint 5 PeVoisin 6 NbSommets 0 NbFaces 0 NbElemDist 20
  Joint 6 PeVoisin 7 NbSommets 14 NbFaces 13 NbElemDist 217
               Total:    NbSommets 95 NbFaces 112 NbElemDist 1583
Writing part 5 into the binary file DOM2_0005.Zones
Writing of 143 nodes.
Writing of 440 elements.
 Construction of part number 6
 Adding of a new joint : 0
 Adding of a new joint : 5
  Number of nodes    : 143
  Number of elements : 440
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 0 NbFaces 0 NbElemDist 44
  Joint 1 PeVoisin 2 NbSommets 18 NbFaces 22 NbElemDist 248
  Joint 2 PeVoisin 5 NbSommets 0 NbFaces 0 NbElemDist 29
  Joint 3 PeVoisin 7 NbSommets 26 NbFaces 36 NbElemDist 367
               Total:    NbSommets 44 NbFaces 58 NbElemDist 688
Writing part 6 into the binary file DOM2_0006.Zones
Writing of 143 nodes.
Writing of 440 elements.
 Construction of part number 7
 Adding of a new joint : 1
 Adding of a new joint : 3
  Number of nodes    : 147
  Number of elements : 442
  Number of joints   : 7
  Joint 0 PeVoisin 0 NbSommets 22 NbFaces 29 NbElemDist 330
  Joint 1 PeVoisin 1 NbSommets 0 NbFaces 0 NbElemDist 2
  Joint 2 PeVoisin 2 NbSommets 17 NbFaces 17 NbElemDist 271
  Joint 3 PeVoisin 3 NbSommets 0 NbFaces 0 NbElemDist 19
  Joint 4 PeVoisin 4 NbSommets 14 NbFaces 13 NbElemDist 276
  Joint 5 PeVoisin 5 NbSommets 14 NbFaces 13 NbElemDist 262
  Joint 6 PeVoisin 6 NbSommets 26 NbFaces 36 NbElemDist 395
               Total:    NbSommets 93 NbFaces 108 NbElemDist 1555
Writing part 7 into the binary file DOM2_0007.Zones
Writing of 147 nodes.
Writing of 442 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
