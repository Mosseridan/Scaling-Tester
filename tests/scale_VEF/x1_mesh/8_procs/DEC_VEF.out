-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF
 
Reading data file 

14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

      3672      13800      13800 [20] [      275:    3672 ]
      1836       7636      10128 [20] [      275:    3672 ]
       918       4368       6456 [20] [      275:    3672 ]
       497       2832       4114 [20] [      275:    3672 ]
       267       1672       3180 [20] [      275:    3672 ]
       145       1022       2482 [20] [      275:    3672 ]
        76        596       1932 [20] [      275:    3672 ]
        40        298       1574 [20] [      275:    3672 ]
        21        136       1286 [20] [      275:    3672 ]
        19        116       1244 [20] [      275:    3672 ]
      1836       6826       6826 [20] [      137:    1836 ]
       918       3704       4990 [20] [      137:    1836 ]
       459       2134       3154 [20] [      137:    1836 ]
       250       1382       1992 [20] [      137:    1836 ]
       136        796       1548 [20] [      137:    1836 ]
        74        484       1180 [20] [      137:    1836 ]
        40        286        942 [20] [      137:    1836 ]
        22        140        740 [20] [      137:    1836 ]
        18        116        718 [20] [      137:    1836 ]
       916       3346       3346 [20] [       68:     916 ]
       458       1800       2430 [20] [       68:     916 ]
       229       1010       1514 [20] [       68:     916 ]
       121        622        926 [20] [       68:     916 ]
        65        348        708 [20] [       68:     916 ]
        37        222        528 [20] [       68:     916 ]
        20        114        420 [20] [       68:     916 ]
       920       3400       3400 [20] [       69:     920 ]
       460       1856       2480 [20] [       69:     920 ]
       230       1056       1560 [20] [       69:     920 ]
       127        704        958 [20] [       69:     920 ]
        70        400        740 [20] [       69:     920 ]
        38        238        562 [20] [       69:     920 ]
        20        132        420 [20] [       69:     920 ]
      1836       6874       6874 [20] [      137:    1836 ]
       918       3792       5038 [20] [      137:    1836 ]
       459       2146       3202 [20] [      137:    1836 ]
       245       1384       1988 [20] [      137:    1836 ]
       133        790       1570 [20] [      137:    1836 ]
        71        484       1182 [20] [      137:    1836 ]
        40        294        938 [20] [      137:    1836 ]
        21        152        748 [20] [      137:    1836 ]
        18        124        722 [20] [      137:    1836 ]
       918       3404       3404 [20] [       68:     918 ]
       459       1836       2486 [20] [       68:     918 ]
       230       1038       1572 [20] [       68:     918 ]
       125        674        964 [20] [       68:     918 ]
        69        390        750 [20] [       68:     918 ]
        36        222        574 [20] [       68:     918 ]
        20        136        442 [20] [       68:     918 ]
       918       3390       3390 [20] [       68:     918 ]
       459       1846       2472 [20] [       68:     918 ]
       230       1066       1558 [20] [       68:     918 ]
       123        630       1030 [20] [       68:     918 ]
        66        376        750 [20] [       68:     918 ]
        36        234        570 [20] [       68:     918 ]
        19        126        430 [20] [       68:     918 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.012
     Coarsening: 		   0.007
            Matching: 			   0.001
            Contract: 			   0.004
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.003
          Refinement: 			   0.002
          Projection: 			   0.001
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:        44616         nmops:         2048  cmop:      0
        num_callocs:          597   num_hallocs:            0
       size_callocs:       628968  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:        44080   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

      3528      13384      13384 [20] [      264:    3528 ]
      1764       7372       9856 [20] [      264:    3528 ]
       882       4228       6328 [20] [      264:    3528 ]
       469       2766       3966 [20] [      264:    3528 ]
       252       1650       3112 [20] [      264:    3528 ]
       137       1032       2426 [20] [      264:    3528 ]
        74        622       1940 [20] [      264:    3528 ]
        39        322       1582 [20] [      264:    3528 ]
        21        150       1274 [20] [      264:    3528 ]
        18        126       1244 [20] [      264:    3528 ]
      1764       6598       6598 [20] [      132:    1764 ]
       882       3600       4834 [20] [      132:    1764 ]
       441       2038       3070 [20] [      132:    1764 ]
       234       1308       1888 [20] [      132:    1764 ]
       128        740       1498 [20] [      132:    1764 ]
        69        464       1134 [20] [      132:    1764 ]
        38        276        908 [20] [      132:    1764 ]
        20        140        710 [20] [      132:    1764 ]
       884       3264       3264 [20] [       66:     884 ]
       442       1762       2380 [20] [       66:     884 ]
       221        974       1496 [20] [       66:     884 ]
       118        628        894 [20] [       66:     884 ]
        62        364        680 [20] [       66:     884 ]
        33        214        510 [20] [       66:     884 ]
        18        116        398 [20] [       66:     884 ]
       880       3244       3244 [20] [       66:     880 ]
       440       1748       2364 [20] [       66:     880 ]
       220        974       1484 [20] [       66:     880 ]
       117        606        906 [20] [       66:     880 ]
        63        348        690 [20] [       66:     880 ]
        33        206        516 [20] [       66:     880 ]
        17        112        402 [20] [       66:     880 ]
      1764       6598       6598 [20] [      132:    1764 ]
       882       3612       4834 [20] [      132:    1764 ]
       441       2038       3070 [20] [      132:    1764 ]
       238       1296       1950 [20] [      132:    1764 ]
       129        736       1514 [20] [      132:    1764 ]
        71        462       1140 [20] [      132:    1764 ]
        37        270        894 [20] [      132:    1764 ]
        19        130        704 [20] [      132:    1764 ]
       882       3258       3258 [20] [       66:     882 ]
       441       1756       2376 [20] [       66:     882 ]
       221        986       1496 [20] [       66:     882 ]
       118        622        918 [20] [       66:     882 ]
        64        358        692 [20] [       66:     882 ]
        36        222        522 [20] [       66:     882 ]
        18        120        404 [20] [       66:     882 ]
       882       3256       3256 [20] [       66:     882 ]
       441       1754       2374 [20] [       66:     882 ]
       221        980       1498 [20] [       66:     882 ]
       119        624        930 [20] [       66:     882 ]
        64        344        716 [20] [       66:     882 ]
        36        220        518 [20] [       66:     882 ]
        20        132        396 [20] [       66:     882 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.014
     Coarsening: 		   0.007
            Matching: 			   0.003
            Contract: 			   0.004
     Initial Partition: 	   0.002
     Uncoarsening: 		   0.005
          Refinement: 			   0.003
          Projection: 			   0.001
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:        42888         nmops:         2048  cmop:      0
        num_callocs:          579   num_hallocs:            0
       size_callocs:       601752  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:        42352   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

15 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 1.10877 s
